assets/css/styleguide.scss

```diff
@@ -321,11 +325,27 @@ $site-top:           124px;
     top: 2em;
     bottom: 2em;
   }
-  padding: 1em $site-margins;
+  padding: 0 $site-margins $site-margins;
   border: 1px solid #eeeeee;
   .is-peripheral {
     opacity: .2;
   }
+
+  fieldset,
+  h6:first-child,
+  h6.usa-heading-alt:first-child {
+    margin-top: 3rem;
+  }
+
+  ul,
+  ol {
+    margin-bottom: 0;
+  }
```

Comment:
This is effecting the actual lists. If you need to change component-wide spacing, it should be done in list.scss. In general we should not be messing with components in styleguide.scss.